Shared/PostList/PostListView.swift

```diff
@@ -126,18 +126,18 @@ struct PostListView: View {
             .frame(height: frameHeight)
             .background(Color(UIColor.systemGray5))
             .overlay(Divider(), alignment: .top)
-            .onReceive(NotificationCenter.default.publisher(for: UIApplication.didBecomeActiveNotification)) { _ in
```

Comment:
Every time the app becomes active, this notification would fire in iOS 17 and navigate us back to the Post list view. Moving it to an `onAppear` modifier below fixes this behaviour will maintaining the list-refresh intent.